osier/technology.py

```diff
@@ -1,17 +1,28 @@
 import unyt
-from unyt import MW, hr
-from unyt import unyt_quantity
+from unyt import MW, hr, kg, km, m
+from unyt import unyt_quantity, unyt_array
 from unyt.exceptions import UnitParseError
 
 import numpy as np
+import pandas as pd
 
 
 _dim_opts = {'time': hr,
              'power': MW,
              'energy': MW * hr,
+             'mass': kg,
+             'length': km,
+             'area': km**2,
+             'volume': m**3,
              'specific_time': hr**-1,
+             'specific_mass': kg**-1,
              'specific_power': MW**-1,
-             'specific_energy': (MW * hr)**-1}
+             'specific_energy': (MW * hr)**-1,
```

Comment:
Isn't specific energy the energy per unit mass? How come these `specific` units don't include mass (aside from `specific_mass`)?

Reply:
*technically* you're correct. The `unyt` library doesn't have currency built in, so I needed some default unit to give to something like "fuel cost" where the dimensions are 1 / energy. I'd be happy to adopt another naming convention if you came up with one!